tokio/src/net/udp.rs

```diff
@@ -2027,16 +2125,19 @@ impl UdpSocket {
         ))))
     )]
     pub fn tos(&self) -> io::Result<u32> {
-        self.as_socket().tos_v4()
+        self.tos_v4()
     }
 
     /// Sets the value for the `IP_TOS` option on this socket.
     ///
     /// This value sets the type-of-service field that is used in every packet
     /// sent from this socket.
     ///
-    /// **NOTE:** On Windows, `IP_TOS` is only supported on [Windows 8+ or
-    /// Windows Server 2012+.](https://docs.microsoft.com/en-us/windows/win32/winsock/ipproto-ip-socket-options)
+    /// # Note
+    ///
+    /// - This may not have any effect on IPv6 sockets.
+    /// - On Windows, `IP_TOS` is only supported on [Windows 8+ or
+    ///   Windows Server 2012+.](https://docs.microsoft.com/en-us/windows/win32/winsock/ipproto-ip-socket-options)
     // https://docs.rs/socket2/0.5.3/src/socket2/socket.rs.html#1446
```

Comment:
Let's update this link to 0.6.1, like in the other related docs.

Reply:
Done.